xbmc/music/tags/TagLoaderTagLib.cpp

```diff
@@ -848,7 +848,7 @@ bool CTagLoaderTagLib::ParseTag(MP4::Tag *mp4, EmbeddedArt *art, CMusicInfoTag&
       SetAlbumArtist(tag, StringListToVectorString(it->second.toStringList()));
     else if (it->first == "soaa")
       SetAlbumArtistSort(tag, StringListToVectorString(it->second.toStringList()));
-    else if (it->first == "----:com.apple.iTunes:ALBUMARTISTS")
+    else if (it->first == "----:com.apple.iTunes:albumartists")
```

Comment:
Just in case someone has managed to write "----:com.apple.iTunes:ALBUMARTISTS" best be backward compatible and test for either
```
else if (it->first == "----:com.apple.iTunes:ALBUMARTISTS" || 
          it->first == "----:com.apple.iTunes:albumartists")
```

Reply:
Done.